### DISCUSSION TITLE
Specify quantity for multi-select dropdowns

### DISCUSSION THREAD

**Discussion by tylermontneyacc** [Budibase/budibase#9134] [Help]
This template is pretty close to what I want: https://budibase.com/forms/templates/employee-equipment-checkout-form/

- One form submission for multiple items
- Users should be able to specify quantity for each item

I thought perhaps I could use a container with a radio button options picker and a numeric field. Once the options picker had a value selected, a new row (of the same) would appear (or a button to add a new row). However, I couldn't figure out how to duplicate that row.

Based on #5658 and #6230, I assume this isn't a feature yet. The first mentions using a multi-picker as a data source (and an "onChange" action) but didn't share the details.

**Comment by melohagan** [comment]
Hey @tylermontneyacc 

As you linked, the dynamic form inputs have not been developed yet. The closest thing you can do at the moment may be to use either [Form steps](https://docs.budibase.com/docs/form-steps) or [Conditional UI](https://docs.budibase.com/docs/conditions)
